### ISSUE TITLE
Can't load linkbutton site; ImportError: No module named 'protocols'

### ISSUE THREAD

**Issue by LimaKilo** [mariusmotea/diyHue#363]
Hi, I tried getting diyHue to run using the automatic installation from the getting started guide. It completed without errors. But I'm not able to reach the linkbutton site to pair with the hue app. Poking around, I tried running "sudo HueEmulator3.py ". This yields

`Traceback (most recent call last):
  File "./HueEmulator3.py", line 30, in <module>
    from protocols import yeelight
ImportError: No module named 'protocols'
`

I suppose something went wrong in the installation script.

**Comment by mariusmotea** [comment]
No, give me one moment to test, the error is in the last commit.

**Comment by mariusmotea** [comment]
Please execute again the installation script, there was a code refactor and installation script was not updated to copy also protocols directory.
